gems/aws-sdk-core/spec/aws/plugins/client_metrics_spec.rb

```diff
@@ -29,15 +29,14 @@ module Plugins
         client.handlers.add(
           ClientMetricsSendPlugin::AttemptHandler,
           step: :sign,
+          priority: 39
```

Comment:
I'm worried about future disconnects in this test (as we had before...).

Stub resposnes removes the CSM handler - which is why we need to add them back in here, right?

Could we call the `add_handlers` function in `ClientMetricsSendPlugin`? like:

```
ClientMetricsSendPlugin.new.add_handlers(client.handlers, client.config)
```

Either that or add constants on the class for the step and priority?